app/players/elves/elf.py

```diff
@@ -0,0 +1,13 @@
+from app.players.player import Player
+
+
+class Elf(Player):
```

Comment:
Since `Elf` does not implement all abstract methods of the `Player` class, `Elf` should also be an abstract class. Inherit it from `ABC` additionally to indicate that. The same relates to the `Dwarf` class as well